ote/src/cljs/ote/views/admin/detected_changes.cljs

```diff
@@ -141,19 +141,15 @@
              [ui/table-row-column (:operator s)]
              [ui/table-row-column (:service-id s)]
              [ui/table-row-column (:service s)]
-             [ui/table-row-column (:type s)]
-             ]))]]]]))
+             [ui/table-row-column (:type s)]]))]]]]))
 
 (defn upload-gtfs [e! app-state]
   [:div
     [:h4 "Lataa palvelulle gtfs tiedosto tietylle päivälle"]
     [form/form
      {:update!   #(e! (admin-controller/->UpdateUploadValues %))
       :footer-fn (fn [data]
-                   [:span
-                    #_ [ui/raised-button {:primary  true
-                                       :on-click #(e! (admin-controller/->ForceRouteHashCalculationForService))
-                                       :label    "Käynnistä gtfs lataus"}]])}
+                   [:span])}
```

Comment:
If footer is not needed, you could just remove :footer-fn. That way there's no redundat dom elements.